xpk.py

```diff
@@ -4857,17 +4896,15 @@ def directory_path_type(value):
 workload_pathways_workload_arguments.add_argument(
     '--proxy-server-image',
     type=str,
-    default='gcr.io/cloud-tpu-v2-images/pathways/pathways-demo:proxy_server',
```

Comment:
lmk if you do want to keep the ability to have a default and we can think through how this would look!

Reply:
I don't think it is super important, as folks would need to replace it during iteration!

Reply:
> during iteration

Can you go into this more? will customers need to replace this often? curious if we should make it simpler if so.